dataverse-webapp/src/main/webapp/filesFragment.xhtml

```diff
@@ -31,7 +31,7 @@
         </div>
     </ui:fragment>
 
-    <ui:fragment rendered="#{!empty datasetFilesTab.workingVersion.fileMetadatas or !empty datasetFilesTab.fileMetadatasSearch or not datasetFilesTab.dataset.hasEverBeenPublished()}">
+    <ui:fragment rendered="#{datasetFilesTab.isFilesTabDisplayable()}">
```

Comment:
When user with assigned good enough permissions (for example superadmin) will search in files something that will not return any results and dataset is under embargo then it will be rendered as:
![Zrzut ekranu z 2020-07-07 10-28-45](https://user-images.githubusercontent.com/8190961/86748874-17059000-c03d-11ea-968e-c233b3e83b86.png)
I think that it would be more secure to add
`<ui:fragment rendered="#{!datasetFilesTab.isFilesTabDisplayable()}">` in fragment one above this (line 15).